nengo_viz/templates/page.html

```diff
@@ -10,7 +10,10 @@
     <script src="static/viz_slider.js"></script>
     <link href="static/viz_slider.css" rel="stylesheet" type="text/css" />
     <script src="static/viz_sim_control.js"></script>
-    <link href="static/viz_sim_control.css" rel="stylesheet" type="text/css" />    
+    <link href="static/viz_sim_control.css" rel="stylesheet" type="text/css" /> 
+    <script src="static/lib/js/jquery-2.1.3.min.js"></script>
+    <script src="static/lib/js/bootstrap.min.js"></script>
+    <link href="static//lib/css/bootstrap.min.css" rel="stylesheet" type="text/css"> 
```

Comment:
Right, so I will move the D3 and interact library.
I'm also thinking of organizing the nengo_viz JS and CSS files as well, because of the organization issues when we add more and more files.